src/components/breadcrumbs/Breadcrumbs.tsx

```diff
@@ -9,7 +9,8 @@ interface BreadcrumbsProps {
 const Breadcrumbs = (props: BreadcrumbsProps) => {
   const routes = [
     { path: "/", breadcrumb: null },
-    { path: "NutritionalInformation", breadcrumb: "Food & Nutrition" },
+    { path: "FoodAndNutrition", breadcrumb: "Food & Nutrition" },
```

Comment:
This path isn't necessary - FoodAndNutrition will get converted to Food & Nutrition in the getBreadcrumbName method

Reply:
Good spot, i had that in before i renamed the routes so it got changed from 'NutritionalInformation' in the renames :)